src/collision/narrow_phase.rs

```diff
@@ -552,6 +608,7 @@ impl<'w, 's, C: AnyCollider> NarrowPhase<'w, 's, C> {
         body2: &RigidBodyQueryReadOnlyItem,
         collider1: &ColliderQueryItem<C>,
         collider2: &ColliderQueryItem<C>,
+        collision_margin: Scalar,
```

Comment:
```suggestion
        collision_margin: CollisionMargin,
```
It's weird to refer to `CollisionMargin` in the docs and then not actually use that type.

Reply:
I don't like having to wrap everything in newtypes unnecessarily so I changed this (and the other similar cases) to `impl Into<CollisionMargin>` and derived `From<Scalar>` for now. Not sure if it's worth it though, since the newtypes don't really give an API benefit (other than explicitness and docs) and it just means that we need to call `.into()` and use derefs in more places internally